integration_tests/test_remote.py

```diff
@@ -0,0 +1,65 @@
+from git_wrapper.repo import GitRepo
+
+
+def test_prune_tags(repo_root):
```

Comment:
I don't have the time to do a full review yet so just a couple of quick notes on this one test! I love to see the work around the integration tests by the way, having this will really help.